src/components/NewMovie/NewMovie.tsx

```diff
@@ -48,6 +102,9 @@ export const NewMovie = () => {
             type="submit"
             data-cy="submit-button"
             className="button is-link"
+            disabled={title.length === 0
+               || imgUrl.length === 0
+               || imdbUrl.length === 0 || imdbId.length === 0}
```

Comment:
```suggestion
            disabled={!title.length
               || !imgUrl.length
               || !imdbUrl.length || !imdbId.length}
```
also pls consider moving this expression to separate const and using here